### PR TITLE
CSM: Clean up.

### PR DESCRIPTION
**Pull request by linbingquan**

### CODE REVIEW

File: examples/jsm/csm/Shader.d.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
-declare namespace _default {
-    export const lights_fragment_begin: string;
-    export const lights_pars_begin: string;
+declare namespace Sharder {
```
Comment (Mugen87) [inline examples/jsm/csm/Shader.d.ts:1]: `Sharder`? Looks like a typo?

Comment (linbingquan) [reply examples/jsm/csm/Shader.d.ts:1]: > `Sharder`? Looks like a typo?

TBH, I think rename as the file name reasonable, if not , I will recover it.

Comment (Mugen87) [reply examples/jsm/csm/Shader.d.ts:1]: Read again, there is an `r` too much in the name.

Comment (linbingquan) [reply examples/jsm/csm/Shader.d.ts:1]: > Read again, there is an `r` too much in the name.

Sorry, you mean I should recover it?

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**